pkg/controller/ingress-shim/sync.go

```diff
@@ -312,13 +312,6 @@ func setIssuerSpecificConfig(crt *cmapi.Certificate, ing *networkingv1beta1.Ingr
 	}
 }
 
-func setCommonName(crt *cmapi.Certificate, ing *networkingv1beta1.Ingress) {
```

Comment:
For visibility, this has been moved to https://github.com/jetstack/cert-manager/blob/51340d0c876147086eea23f3c827906cefdad8e4/pkg/controller/ingress-shim/helper.go#L36